manifests/base/v2alpha1/crds/kubeflow.org_clustertrainingruntimes.yaml

```diff
@@ -0,0 +1,9275 @@
+---
```

Comment:
I am wondering are what are advantages for the CEL validation if we have validation webhook ?
I noticed that if CRD becomes to large it can cause issues. For example: https://github.com/kserve/kserve/issues/3487
cc @terrytangyuan @kubeflow/wg-training-leads

Reply:
That's not a problem. We can avoid the problem with `--server-side` option.
We should introduce the CEL validation to reduce Kube-API server loads.

In the big cluster, many webhook validations are a big deal for performance issues. Specifically, kube-apiserver and etcd are so busy in batch clusters with hundreds and thousands of Jobs.

For UX, CEL validation could be reduce kubectl / SDK response time since the kube-apiserver does not need to call external webhook servers, the kubeapi-server could just evaluate incoming Job objects inside of kube-apiserver.

Reply:
Make sense! Let's see if our CRD can go too-large even with server side option.